### PR TITLE
Adds second log

### PR DESCRIPTION
**Pull request by frm**

### CODE REVIEW

File: 201602_002.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+# 201602 002
+
+A couple of days after the first logged event, Fernando, Miguel and Pedro are in a Ruby meetup. This time there were live demos.
```
Comment (pfac) [inline 201602_002.md:3]: Don't spoil the live demos part, the log already starts with it :stuck_out_tongue:

File: 201602_002.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+## 11
+
+Miguel just found a bug. It's Pedro's fault. Now it's Miguel's fault. Now it's Pedro's fault. Now it's the designer's fault.
+
+## 12
+
+It's Pedro's turn up at bat.
+
+Pedro does not want to offend the public, but offends the public by saying he wanted a more experienced public.
```
Comment (pfac) [inline 201602_002.md]: I feel this should be a separate entry.

File: 201602_002.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+## 13
+
+Pedro blames the computer for spontaneously advancing his slides. Pedro doesn't know how computers work.
```
Comment (pfac) [inline 201602_002.md]: The second sentence should be something like `Pedro doesn't know how to computer`, following our initial style. And it should be on a different line.

File: 201602_002.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+Pedro changes focus to his terminal and starts typing. Pedro is doomed.
+
+## 22
+
+Pedro is now live coding.
+
+## 23
+
+Pedro thinks by showing "the simplest possible example" he's immune to the curses of live demos.
```
Comment (pfac) [inline 201602_002.md]: I feel this deserves a `Oh silly Pedro` sitcom-style comment.

File: 201602_002.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+
+## 24
+
+Oh God, it's the calculator from the last log. You can't make this up.
+
+## 25
+
+First run, all tests passed. Clearly the tests are wrong.
+
+
```
Comment (pfac) [inline 201602_002.md:109]: Extra line

**Comment by pfac** [comment]
Revise missing colons. Also, the first log did not place an empty line between the entry number and its text. We should keep the style consistent at least through the first pair of logs.

File: 201602_002.md (hunk 1 above)
Comment (frm) [inline 201602_002.md:3]: How about "Of course neither one of them spoke about Ruby."?

**Pull request merged**